storage/store.go

```diff
@@ -657,6 +669,7 @@ func (sc *StoreContext) setDefaults() {
 	}
 
 	raftElectionTimeout := time.Duration(sc.RaftElectionTimeoutTicks) * sc.RaftTickInterval
+	sc.concurrentSnapshotLimit = 5
```

Comment:
Is there a rationale behind this setting? I would have guessed that we'd want this set to `1`.